# Goal - Either a set of target joints OR a target point.
#        (If both are given, the joints will take precedent and the point will be ignored.)
float64[] target_joints         # [shoulder pan, shoulder tilt, elbow tilt, wrist rotate]
geometry_msgs/PointStamped target_point
---
# Result - Success result & final arm position
bool success
float64[] arm_position          # [shoulder pan, shoulder tilt, elbow tilt, wrist rotate]
---
# Feedback - Current arm position
float64[] arm_position          # [shoulder pan, shoulder tilt, elbow tilt, wrist rotate]
